pkg/template/servicebroker/catalog.go

```diff
@@ -109,7 +117,11 @@ func serviceFromTemplate(template *templateapi.Template) *api.Service {
 	}
 }
 
+// Catalog returns our service catalog (one service per OpenShift template in
+// configured namespace(s)).
 func (b *Broker) Catalog() *api.Response {
+	glog.V(4).Infof("Template service broker: Catalog")
```

Comment:
have added this to the service broker startup log message - unless you really want it logged on every time a catalog call is made (and loglevel >= 4)?